App/screens/onboarding/ImportMnemonicScreen/index.js

```diff
@@ -3,6 +3,10 @@ import NavigationProvider from '../../../NavigationProvider';
 import BaseImportMnemonicScreen from './base';
 
 export default class ImportMnemonicScreen extends React.Component {
+  static navigationOptions = { 
```

Comment:
Seems like this is for react navigation, not react-native-navigation?

Reply:
Oh, I should use navigatorStyle on this case... Thanks for reminder!